### ISSUE TITLE
Full file source?

### ISSUE THREAD

**Issue by du291** [twitter/scalding#1225]
Hi, I'm trying to slurp whole files as pipe elements. Rationale behind this is, I have JSON files but they are pretty printed, rather than one JSON per line. I also could imagine other uses, like loading PNG files. I've tried this:

case class Boo(p:String) extends FixedPathSource(p)

class WholeFileReader(args : Args) extends Job(args) {
    Boo(args("input")).write(TextLine(args("output")))
}

But I get exception com.twitter.scalding.ModeException: Cascading Hadoop mode not supported for: my.package.BooWrappedArray(/path/to/files)

I suppose that's because I did not override the defaults with any Scheme. But I'm not an expert on scalding/cascading internals so I'm really lost. How should I go about this? Or is there already a Source that would allow me to do this?

**Comment by ianoc** [comment]
Yeah you may need a scheme but you will definitely need a fileinputformat
that may already exist on the web as it's an Hadoop thing. Probably not an
easy task if your not too familiar with Hadoop either

**Comment by du291** [comment]
I will look through the web. Any tips or guidance would be appreciated.
